compiler/powershell.vim

```diff
@@ -0,0 +1,63 @@
+" Compiler:	powershell
+" Run ps1 scripts in powershell and process their output. Quickly jump through
+" stack traces and see script output in the quickfix.
+
+if exists("current_compiler")
+  finish
+endif
+let current_compiler = "powershell"
+
+if exists(":CompilerSet") != 2		" older Vim always used :setlocal
+  command -nargs=* CompilerSet setlocal <args>
+endif
+
+let s:cpo_save = &cpo
+set cpo-=C
+
+" Show CategoryInfo, FullyQualifiedErrorId, etc?
+let g:ps1_efm_show_error_categories = get(g:, 'ps1_efm_show_error_categories', 0)
+
+" Use absolute path because powershell requires explicit relative paths
+" (./file.ps1 is okay, but # expands to file.ps1)
+CompilerSet makeprg=powershell\ %:p
+
+" Parse file, line, char from callstacks:
+"     Write-Ouput : The term 'Write-Ouput' is not recognized as the name of a
+"     cmdlet, function, script file, or operable program. Check the spelling
+"     of the name, or if a path was included, verify that the path is correct
+"     and try again.
+"     At C:\script.ps1:11 char:5
+"     +     Write-Ouput $content
+"     +     ~~~~~~~~~~~
+"         + CategoryInfo          : ObjectNotFound: (Write-Ouput:String) [], CommandNotFoundException
+"         + FullyQualifiedErrorId : CommandNotFoundException
+"
+" Error message may span multiple lines (it's wrapped to width of terminal).
+CompilerSet errorformat=%+G+%m
```

Comment:
This will change with pwsh 7. I recommending reading up on the change. There is a way to restore this original so this could always work, or can some lines specified below be optional?

Reply:
...or maybe you can just add it as another option like I see below.

Reply:
You mean that the error may span multiple lines? This errorformat should handle single or multi line errors.

Reply:
The format will change. See https://devblogs.microsoft.com/powershell/powershell-7-preview-5/#conciseview-for-errors (not actually an anchor, it turns out; search for "conciseview for errors").

Reply:
It's so different that it should be a different compiler.

Reply:
Woah.
```
Get-ChildItem: in /Users/Username/GitHub/Errorview/script.test.ps1
Line |
15   | Get-ChildItem -Path c:\notreal
     |                     ^^^ Cannot find path 'C:\notreal' because it does not exist.
```
It's so different that maybe it should be a different compiler. pwshconcise.vim? It's possible you could add the two efm together, but since they're so different I'd expect them to conflict.

Maybe the compiler script should never invoke with concise (regardless of user's default) since the old format is easier to parse. It doesn't look like they're eliminating the old format?